### ISSUE TITLE
[FEATURE] A way to backup a server or a node

### ISSUE THREAD

**Issue by r0xANDt0l** [PufferPanel/PufferPanel#1180]
It would be really nice to be able to backup a server or an entire node and keep a save on your server or locally on your pc. And have the ability to save it in several file formats, like in .zip, .tar.lz4, etc...

**Comment by pierrbt** [comment]
Yes it should be cool to backup the server automatically at a certain hour. You could also all `lbzip2` compression which is one of the best.

**Comment by shannonr** [comment]
Yes please, if we could have backup templates like we have install templates that would be amazing.

For example I'd be looking to run hourly backups of my Minecraft Server, I'd need to send commands on the console (stop save, save all etc) zip up the minecraft directory (would be good to have excludes as well) then run another command. Then the backup could run at a chosen interval and be kept for a certain length of time e.g. save every hour and keep them for a rolling 24 hours.

I'm sure other users would have other use cases.
